src/components/Claim/ClaimCard.tsx

```diff
@@ -79,15 +62,21 @@ const ClaimCard = ({ personality, claim, collapsed = true }) => {
             });
             setClaimContent(textContent.trim());
         };
-        if (isSpeech) {
+        if (isSpeech || isGenerativeInformation) {
```

Comment:
suggestion: transform this into a function, in the future we want a better way of identifying the reason to trigger the `CreateFirstParagraph` function.